vars/longevityPipeline.groovy

```diff
@@ -85,7 +91,9 @@ def call(Map pipelineParams) {
                                         exit 1
                                     fi
 
-                                    export SCT_FAILURE_POST_BEHAVIOR="${pipelineParams.params.get('post_behaviour', '')}"
+                                    export SCT_POST_BEHAVIOR_DB_NODES="${pipelineParams.params.get('post_behaviour_db_nodes', '')}"
```

Comment:
you need to go over all pipelines and update this 
i.e. lookup 'SCT_FAILURE_POST_BEHAVIOR' it's there in all pipelines

Reply:
Fixed